### DISCUSSION TITLE
I can’t figure out why?

### DISCUSSION THREAD

**Discussion by lishofaramazan** [zkSync-Community-Hub/zksync-developers#598] [General] [answered]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

Good afternoon. I decided to check the checker for the presence of a drop, and was shocked, to put it mildly, that my wallet did not qualify for a drop. I can't understand why? I have 1 wallet, I repeat, I have 1 wallet, I did not use account farms as is now fashionable, I spent a lot of time and money, I am the owner of NFT Omnibus Libertas, my wallet is in the top 0.2% in terms of activity rating. I think I did a very good job, why did the team treat my work this way? Please look into this problem and I think there was an error, thank you!
https://nftcopilot.com/zksync-rank-check?address=0x9b1711D6cdc820522Db3c9FeeB2C1354Ca2F92d6
My wallet 0x9b1711D6cdc820522Db3c9FeeB2C1354Ca2F92d6

**Comment by 0xsisyfos** [comment, marked as answer]
Hello,

Thank you for reaching out.

The ZK Token FAQ provides additional details on eligibility criteria: https://docs.zknation.io/zk-token/zk-token-faq

In particular, question 3 ("I met some of the eligibility criteria, but did not get an allocation, why not?") answers your question directly.

As a note, this space if for developer questions, please join the Discord for further support: https://join.zksync.dev/
